### ISSUE TITLE
NTR: Pancreatic biopsy: carcinoid tumor (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9122]
New term request Pancreatic biopsy: carcinoid tumor (elthat_100428052222):   UMLS: 0345933

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000233
- label: Carcinoid tumor of the pancreas
- parents: [Neoplasm of the pancreas (HP:0002894)](https://hpo.jax.org/app/browse/term/HP:0002894)
- pmid: PMID:28454406
- def: A carcinoid tumor of the pancreas (CTP) is a rare neuroendocrine tumor arising from the serotonin-secreting cells (enterochromaffin or Kultscitsky's cells).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9122
- synonyms:
